Src/AmrCore/AMReX_AmrMesh.H

```diff
@@ -253,13 +257,20 @@ public:
 
     [[nodiscard]] long CountCells (int lev) noexcept;
 
+    [[nodiscard]] virtual DistributionMapping MakeDistributionMap (int lev, BoxArray const& ba);
+
 protected:
 
     int finest_level;    //!< Current finest level.
     Vector<Geometry>            geom;
     Vector<DistributionMapping> dmap;
     Vector<BoxArray>            grids;
 
+#ifdef AMREX_USE_BITTREE
+    bool use_bittree;
+    std::shared_ptr<bittree::BittreeAmr> btmesh;
```

Comment:
```suggestion
    std::unique_ptr<bittree::BittreeAmr> btmesh;
```
We could use unique_ptr instead.